modules/installation-vsphere-machines.adoc

```diff
@@ -165,10 +164,14 @@ $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameser
 $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
 ----
 +
+[comment]
+--
+//Need to replace with alternative text to give a possible use case of diagnosing cluster performance issues. Possible a resource pool configuration.
 *** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
 **** Memory reservation value must be equal to its configured memory size.
-**** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
-*** Click *Edit Configuration*, and on the *Configuration Parameters* window, search the list of available parameters for steal clock accounting (`stealclock.enable`). If it is available, set its value to `TRUE`. Enabling steal clock accounting can help with troubleshooting cluster issues.
+--
+*** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
```

Comment:
1. Latency sensitivity should be completely removed, we should not be even optionally recommending this setting.
2. IMHO we should not be suggesting to a user to set reservations directly on a guest, this should be in a resource pool. Setting at the guest level should be a last resort.
3. I do not believe we should be telling users what the allocation for memory and cpu reservations should be. This has to be configured by the customers' vSphere administrator based on the physical design and the guests running in that cluster. Those prescribed settings could have serious impacts to the vSphere enviornment though not as severe as latency sensitivity to high. Anything we state should be suggestions only and not *must*.

Reply:
Thanks, @jcpowermac . @jcpowermac  and @Timdawson264 . I removed some content and marked section 9.f as optional. I also adding a disclaimer that places the onus on a cluster admin to ensure they configure their cluster accordingly. Please let me know your thoughts. 

I'm not sure how granular I need to be with the resource pool suggestions or if the resource pool could be a focal point for removed steps?